packages/blocks/src/__internal__/service/components.ts

```diff
@@ -23,7 +23,21 @@ export function BlockElement(
   edgeless = false
 ) {
   switch (model.flavour) {
-    case 'affine:paragraph':
+    case 'affine:paragraph': {
+      const rowHost = { ...host };
+      const parentModel = model.page.getParent(model);
+      if (parentModel?.flavour === 'affine:database') {
+        rowHost.flavour = 'affine:database';
```

Comment:
You should pass in a different host from database block

Reply:
Guess we can pass a new host here

https://github.com/toeverything/blocksuite/blob/master/packages/blocks/src/database-block/database-block.ts#L150

Reply:
oh, more clearly here.